src/fftvis/wrapper.py

```diff
@@ -174,6 +176,15 @@ def simulate_vis(
         for the CoordinateRotationERFA method, there is the parameter ``update_bcrs_every``,
         which should be a time in seconds, for which larger values speed up the computation.
         See the documentation for the CoordinateRotation classes in matvis for more information.
+    skew_to_cartesian : np.ndarray, default = None
+        Linear change of basis matrix that maps antenna positions expressed in a skewed/oblique grid 
+        into the standard Cartesian frame. This is used to convert the antenna positions from the skewed grid
+        to a Cartesian grid that is compatible with a more efficient NUFFT algorithm. By default fftvis checks
+        if the antenna positions are on a regular or hexagonal grid, but other grid types are not automatically
+        supported. The matrix should be of shape (3, 3).
+    force_use_type3: bool, default = False
+        Whether to force the use of type 3 NUFFT. If False, type 3 will only be used
+        if the array can be defined in a regular grid.
```

Comment:
Isn't this the wrong way around? Type 3 will be used if the array *cannot* be defined on a regular grid?

Reply:
Yeah, that's right. I'll change that. Thanks for the catch.